base/untar.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Download and extract tarball archive from the network.
+
+set -ex
+
+case "$1" in
+    -r) dest=/ ;;
+    -l) dest=. ;;
+    *) >&2 echo "Invalid extraction mode: must be either root (-r) or local (-l)."
+       exit 1;
+    ;;
+esac
+
+shift
+
+urls=$@
+
+for url in $urls; do
```

Comment:
This can be done as `for url; do`, which uses the `$@` arguments automatically

Reply:
Nice! I didn't know about that. Thanks!